bin/generate-jwks

```diff
@@ -1,24 +1,33 @@
 #!/usr/bin/env ruby
 
 # This script is meant to be run locally on a dev machine (not a server), and it
-# is used to generate new JSON Web Keys (JWK) to be added to the
-# JSON Web Key Set (JWKS) exposed at /oauth/jwks. It generates both the public
-# key and the private key. The public key will be added to the JWKS in
-# dashboard/public/oauth/jwks.json. It will print the private key and
-# key id (kid) that is to be used for updating the secret at
-# CDO.jwk_private_key_data.
-
+# is used to generate new JSON Web Keys (JWKS). It generates both the public key
+# and the private key. The public key will be formated in YAML, to simplify adding
+# it to the approprate config files (see below). The private key will be formated
+# as JSON, which is how it should be stored in Secrets Manager.
+#
+# Here are the locations the public key should be added
+# - For Production, the public key should be added to config/production.yml.erb
+# - For Non-Prod, the public key should be added to config.yml.erb
```

Comment:
Might be worth incorporating a bit of an explanation of these config files into this instruction; something like,

```suggestion
# The public key used by default should be defined in `config.yml.erb`, for use in
# non-production environments. For the production environment, a production-specific
# public key override should be added to `config/production.yml.erb`.
```